docs/src/solutions.md

```diff
@@ -74,21 +63,17 @@ JuMP.dual_status
 MOI.ResultStatusCode
 ```
 
-TODO
-same as above
-
-TODO
-do we add the tables from MOI? (they are very useful)
+Common status situations are described in [`MathOptInterface`'s docs](http://www.juliaopt.org/MathOptInterface.jl/stable/apimanual/#Common-status-situations-1).
 
 ## Obtaining solutions
 
 Solution are typically queried with the functions [`JuMP.value`](@ref) and [`JuMP.dual`](@ref).
 These functions should be applied to references: [`VariableRef`](@ref) or a [`ConstraintRef`](@ref).
 Depending on the type of variable and constraint associated with the reference the
-return of the function, it can be a scalar or an array.
+return of the function can be a scalar, an array or an even more complex container (see [`AbstractShape`](@ref) and [`dual_shape`](@ref)).
```

Comment:
I wouldn't say `container`, we use container for Array, Containers.DenseAxisArray and Containers.SparseAxisArray

Reply:
I see, you suggest using `shape` instead?

Reply:
I would say "is typically a scalar or an array but it can be an arbitrary object (see ...)"